src/test/lib/operators/union_positions_test.cpp

```diff
@@ -55,7 +56,7 @@ TEST_F(UnionPositionsTest, SelfUnionSimple) {
   ASSERT_EQ(table_scan_a_op->get_output()->row_count(), 4u);
   ASSERT_EQ(table_scan_b_op->get_output()->row_count(), 4u);
 
-  auto union_unique_op = std::make_shared<UnionPositions>(table_scan_a_op, table_scan_a_op);
+  auto union_unique_op = std::make_shared<UnionPositions>(table_scan_a_op, table_scan_b_op);
```

Comment:
That is actually correcting an old mistake, isn't it?

Reply:
correct